sources/web/datalab/polymer/polymer.json

```diff
@@ -9,7 +9,10 @@
     "components/datalab-sidebar/datalab-sidebar.html",
```

Comment:
If you like, we could also clean these up with this change. For example, `datalab-editor`, `datalab-sidebar`, and `datalab-toolbar` can all be removed.